tests/cupy_tests/sorting_tests/test_sort.py

```diff
@@ -11,11 +12,15 @@ class TestSort(unittest.TestCase):
 
     # Test ranks
 
+    # TODO(niboshi): Fix xfail
+    @pytest.mark.xfail(strict=True, reason='Explicit error types required')
```

Comment:
`strict=True` requires further care.  In fact, XPASS with numpy 1.12.  (`AxisError` is since 1.13.  xfail by a different reason with numpy 1.9.)

Reply:
OK, removed in c7f558e88c5136743f8c7c815201dbbf6af9787c.